capacitor.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appId": "com.example.artfunder",
+  "appName": "Artfunder",
+  "bundledWebRuntime": false,
+  "npmClient": "npm",
+  "webDir": "build",
```

Comment:
i think it should www instead of build because capacitator will give problems otherwise